### DISCUSSION TITLE
如何提升版面分析模型精度

### DISCUSSION THREAD

**Discussion by S130111** [PaddlePaddle/PaddleOCR#12254] [Q&A]
在CDLA数据集上进行训练测试，效果不是很理想，有人能帮解决一下吗
![1693289997197](https://github.com/PaddlePaddle/PaddleOCR/assets/45257011/d16bc495-01fd-4630-b4bc-894269cbece7)

**Comment by shiyutang** [comment]
ace-issue-solver response：https://github.com/shiyutang/Acceleratedcpp/issues/10
可以持续向其提问
